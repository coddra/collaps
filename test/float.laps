0.5 .
0.0003e+3 .
0xAp-2 .
4.0 - 3.2 .
