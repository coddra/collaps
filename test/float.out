0.500000
0.300000
2.500000
0.800000
